pkg/get/tools.go

```diff
@@ -121,7 +121,7 @@ helmfile_{{$os}}_{{$arch}}{{$ext}}`,
 {{$os = "windows"}}
 {{- end -}}
 
-https://storage.googleapis.com/kubernetes-release/release/v{{.Version}}/bin/{{$os}}/{{$arch}}/kubectl{{$ext}}`})
+https://storage.googleapis.com/kubernetes-release/release/{{.Version}}/bin/{{$os}}/{{$arch}}/kubectl{{$ext}}`})
```

Comment:
I can't see the difference. We had:

v + 1.20.0

Then the change moves the v to the Version string, so it still prints v1.20.0

Reply:
Here's an example of what I mean for `doctl`:

https://github.com/alexellis/arkade/blob/master/pkg/get/tools.go#L406

With the version passed in without a `v` prefix

https://github.com/alexellis/arkade/blob/master/pkg/get/tools.go#L382

Perhaps we could have a `v` where that corresponds to the actual GitHub release, and then a Go templates function which can add the prefix if missing, but which won't duplicate it?

Reply:
OK I can see that this was broken in #430 in a PR by @dirien 

https://github.com/alexellis/arkade/pull/430/files

This is on me for not picking it up. Sorry about that.